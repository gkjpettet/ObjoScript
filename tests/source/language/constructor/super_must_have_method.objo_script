class A {}

class B is A {
  constructor() {
    super // expect error as method call expected.
  }
}