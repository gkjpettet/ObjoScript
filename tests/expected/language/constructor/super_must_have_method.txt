Expected an opening parenthesis or a dot after `super`.